src/coreservices.h

```diff
@@ -122,6 +121,7 @@ class CoreServices : public QObject {
     void initializeSettings();
     void initializeScreensaverManager();
     void initializeLogging();
+    void shutdown();
```

Comment:
The deceleration of this method should be moved closed to that of initialize(). Please add documentation that it *undoes* the result of initialize().